packages/snet_cli/test/functional_tests/mint/mint.py

```diff
@@ -0,0 +1,62 @@
+from packages.snet_cli.snet.snet_cli.utils.utils import get_web3, get_contract_object
+from web3.gas_strategies.time_based import medium_gas_price_strategy
+
+TRANSACTION_TIMEOUT = 500
+DEFAULT_GAS = 300000
+HTTP_PROVIDER = "http://localhost:8545"
+
+wallet_address_1 = "0x592E3C0f3B038A0D673F19a18a773F993d4b2610"
```

Comment:
Lets remove private keys from the code.

Reply:
@rajeevbbqq , you can create a random pvt key and public key on the  fly , tools avb to do so

Reply:
@raamb & @anandrgitnirman hardcoded keys are also referred to in other test cases, and other test cases will fail if we create a key on the fly.

Reply:
Ok sure. Lets go ahead with this for now